# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../partimage/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Naurp partimage-0.6.7-orig/src/shared/net.h partimage-0.6.7/src/shared/net.h
--- partimage-0.6.7-orig/src/shared/net.h	2008-02-03 15:57:55.000000000 -0600
+++ partimage-0.6.7/src/shared/net.h	2008-07-16 20:04:02.000000000 -0600
@@ -28,7 +28,7 @@
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <arpa/inet.h>
-#include <iostream.h>
+#include <iostream>
 #include <pthread.h>
 
 #include "pathnames.h"
diff -Naurp partimage-0.6.7-orig/src/server/partimaged.cpp partimage-0.6.7/src/server/partimaged.cpp
--- partimage-0.6.7-orig/src/server/partimaged.cpp	2008-02-03 15:57:53.000000000 -0600
+++ partimage-0.6.7/src/server/partimaged.cpp	2008-07-16 20:10:04.000000000 -0600
@@ -22,7 +22,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
-#include <iostream.h>
+#include <iostream>
 
 #include "net.h"
 #include "netserver.h"
diff -Naurp partimage-0.6.7-orig/src/server/partimaged-main.cpp partimage-0.6.7/src/server/partimaged-main.cpp
--- partimage-0.6.7-orig/src/server/partimaged-main.cpp	2008-02-03 15:57:53.000000000 -0600
+++ partimage-0.6.7/src/server/partimaged-main.cpp	2008-07-16 20:10:45.000000000 -0600
@@ -27,7 +27,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
-#include <iostream.h>
+#include <iostream>
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <signal.h>
